Remotion/ObjectBinding/Web.Test/IndividualControlTests/BocListUserControl.ascx.cs

```diff
@@ -392,6 +400,186 @@ private void ChildrenListRemoveRowsButton_Click (object sender, EventArgs e)
+                        new ValidatedProperty(firstChild, personFullNameProperty)
+                    },
+                    "Bad first child names",
+                    "Localized bad first child names"),
+                // Last child failure (invisible)
+                new ObjectValidationFailure(
+                    lastChild,
+                    "Bad last child",
+                    "Localized bad last child"),
+                new PropertyValidationFailure(
+                    lastChild,
+                    personPartnerProperty,
+                    lastChild.Partner,
+                    "Bad last child.Partner",
+                    "Localized bad last child.Partner"),
+                new ObjectValidationFailure(
+                    lastChild,
+                    new[]
+                    {
+                        new ValidatedProperty(lastChild, personLastNameProperty),
+                    },
+                    "Bad last child name",
+                    "Localized bad last child name"),
+                new ObjectValidationFailure(
+                    lastChild,
+                    new[]
+                    {
+                        new ValidatedProperty(lastChild, personLastNameProperty),
+                        new ValidatedProperty(lastChild, personFullNameProperty)
+                    },
+                    "Bad last child names",
+                    "Localized bad last child names"),
+            }));
+
+    PrepareValidation();
+    FormGridManager.PrepareValidation();
+    DataSourceValidationResultDispatchingValidator.DispatchValidationFailures(validationResult);
+    DataSourceValidationResultDispatchingValidator.Validate();
+
+    UnhandledValidationErrorsLabel.Text = string.Join(
+        Environment.NewLine,
+        validationResult.GetUnhandledValidationFailures().Select(FormatValidationFailure));
+
+    static string FormatValidationFailure (ValidationFailure failure)
+    {
+      return failure switch
+      {
+        PropertyValidationFailure propertyValidationFailure => $"Unhandled property validation failure '{propertyValidationFailure.ErrorMessage}'"
+                                                               + $" for property '{propertyValidationFailure.ValidatedProperty.Name}'"
+                                                               + $" on object '{propertyValidationFailure.ValidatedObject}'.",
+        ObjectValidationFailure objectValidationFailure => $"Unhandled object validation failure '{objectValidationFailure.ErrorMessage}'"
+                                                           + $" for properties '{string.Join(", ", objectValidationFailure.ValidatedProperties.Select(e => e.Property.Name))}'"
+                                                           + $" on object '{objectValidationFailure.ValidatedObject}'.",
+        _ => $"Unhandled property validation failure '{failure.ErrorMessage}' on object '{failure.ValidatedObject}'."
```

Comment:
We don't yet have a pre-existing code style for this particular case. Based on other rules, I think there should be a line break after the arrow (most likely it is "after the arrow" and not "before the arrow", given how we format anonymous method declarations with a statement-block body.